cmd/thanos/query_frontend.go

```diff
@@ -175,12 +174,6 @@ func runQueryFrontend(
 		return errors.Wrap(err, "error validating the config")
 	}
 
-	fe, err := cortexfrontendv1.New(cortexfrontendv1.Config{}, nil, logger, reg)
```

Comment:
Wasn't used by Thanos. Thanos use the tripperware below. Basically this code was doing nothing.

Reply:
cc @yeya24 🤗

Reply:
Yes, it is not used so it is safe to remove it. Thanks @pracucci !